### PR TITLE
:technologist: add Docker Compose file with web and mailhog services

### PR DESCRIPTION
**Pull request by issa-diallo**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+services:
+
+  web:
+    build:
+      context: .
+      args:
+        - PORT=8000
+        - VERSION=${VERSION:-latest}
+    ports:
+      - "${WEB_PORT:-8000}:8000"
```
Comment (AndreMiras) [inline docker-compose.yml:12]: What is this variable for?

**Review by AndreMiras — COMMENTED** [review]
Nice to have a docker compose, thanks.
Left a comment

File: docker-compose.yml (hunk 1 above)
Comment (issa-diallo) [reply docker-compose.yml]: there is no variable, so I changed the syntax

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - EMAIL_PORT=1025
+      - EMAIL_HOST_USER
+      - EMAIL_HOST_PASSWORD
+      - DATABASE_ENGINE
+      - DATABASE_NAME
+      - DATABASE_USER
+      - DATABASE_PASSWORD
+      - DATABASE_HOST
+      - DATABASE_PORT
+      - DJANGO_DEBUG=True
```
Comment (AndreMiras) [inline docker-compose.yml:24]: where is this used?

Comment (issa-diallo) [reply docker-compose.yml]: I've just deleted

**Review by AndreMiras — APPROVED** [review]

**Review by AndreMiras — APPROVED** [review]

**Pull request merged**